DeepCat.js

```diff
@@ -139,14 +139,14 @@
 	}
 
 	function getAjaxRequest( searchTerm, searchTermNum ) {
-		var categoryString = DeepCat.extractDeepCatCategory( searchTerm ),
-			userParameter = {
+		({  categoryString, depth } = DeepCat.extractDeepCatCategory( searchTerm ));
```

Comment:
I'm not sure if this was a good idea. If a user requests 1, he should get 1. If he requests 0, he should get 0. The JavaScript code could even go so far and replace this with a trivial `incategory:` without asking the DeepCat server.